modules/map/utils/findShapesInRadius.ts

```diff
@@ -0,0 +1,54 @@
+import { Feature, MultiPolygon, Polygon, Position } from 'geojson'
+
+import { MapUdrZone } from '@/modules/map/types'
+
+function getDistance(point1: Position, point2: Position): number {
+  const [x1, y1] = point1
+  const [x2, y2] = point2
+
+  return Math.hypot(x2 - x1, y2 - y1)
+}
+
+export const findShapesInRadius = (
```

Comment:
Some TSDoc would be more than welcome. At least, what is the unit of radius - meters, kilometers, something else?